app/wasm/test/messages_test.go

```diff
@@ -64,6 +64,110 @@ func TestCreateDenom(t *testing.T) {
 
 }
 
+func TestChangeAdmin(t *testing.T) {
+	const validDenom = "validdenom"
+
+	actor := RandomAccountAddress()
```

Comment:
Lets rename this to token creator?

So then we distinguish whose executing the change admin function from the creator more clearly?

Reply:
Renamed